README.md

```diff
@@ -1,23 +1,42 @@
-Victor CLI
-==================================================
+# Victor CLI
 
 [![Gem Version](https://badge.fury.io/rb/victor-cli.svg)](https://badge.fury.io/rb/victor-cli)
 [![Build Status](https://github.com/DannyBen/victor-cli/workflows/Test/badge.svg)](https://github.com/DannyBen/victor-cli/actions?query=workflow%3ATest)
+[![Maintainability](https://api.codeclimate.com/v1/badges/a5fa93e41f935e3148a9/maintainability)](https://codeclimate.com/github/DannyBen/victor-cli/maintainability)
 
 ---
 
-CLI for Victor, the SVG Library
+Command line interface for Victor, the SVG Library.
```

Comment:
Perhaps there could be a link over to the repo for `victor`?

Reply:
Yes. Adding.